tests/system/test_database_api.py

```diff
@@ -73,6 +74,34 @@ def test_create_database(shared_instance, databases_to_delete, database_dialect)
     assert temp_db.name in database_ids
 
 
+def test_database_binding_of_pool(
```

Comment:
Add test case for FixedSizePool as well.

Reply:
Added